roles/consensus/templates/testnet/app.toml.j2

```diff
@@ -0,0 +1,181 @@
+# It has no bearing on application state pruning which is determined by the
+# "pruning-*" configurations.
+#
+# Note: CometBFT block pruning is dependent on this parameter in conjunction
+# with the unbonding (safety threshold) period, state pruning and state sync
+# snapshot parameters to determine the correct minimum value of
+# ResponseCommit.RetainHeight.
+min-retain-blocks = 0
+
+# InterBlockCache enables inter-block caching.
+inter-block-cache = true
+
+# IavlCacheSize set the size of the iavl tree cache (in number of nodes).
+iavl-cache-size = 2500
+
+# IAVLDisableFastNode enables or disables the fast node feature of IAVL. 
+# Default is false.
+iavl-disable-fastnode = true
+
+
+###############################################################################
+###                         Telemetry Configuration                         ###
+###############################################################################
+
+[telemetry]
+
+# Prefixed with keys to separate services.
+service-name = ""
+
+# Enabled enables the application telemetry functionality. When enabled,
+# an in-memory sink is also enabled by default. Operators may also enabled
+# other sinks such as Prometheus.
+enabled = true
+
+# Enable prefixing gauge values with hostname.
+enable-hostname = false
+
+# Enable adding hostname to labels.
+enable-hostname-label = false
+
+# Enable adding service to labels.
+enable-service-label = false
+
+# PrometheusRetentionTime, when positive, enables a Prometheus metrics sink.
+prometheus-retention-time = 0
+
+# GlobalLabels defines a global set of name/value label tuples applied to all
+# metrics emitted using the wrapper functions defined in telemetry package.
+#
+# Example:
+# [["chain_id", "cosmoshub-1"]]
+global-labels = [
+]
+
+# MetricsSink defines the type of metrics sink to use.
+metrics-sink = ""
+
+# StatsdAddr defines the address of a statsd server to send metrics to.
+# Only utilized if MetricsSink is set to "statsd" or "dogstatsd".
+statsd-addr = ""
+
+# DatadogHostname defines the hostname to use when emitting metrics to
+# Datadog. Only utilized if MetricsSink is set to "dogstatsd".
+datadog-hostname = ""
+
+###############################################################################
+###                                BeaconKit                                ###
+###############################################################################
+
+[beacon-kit.engine]
+# HTTP url of the execution client JSON-RPC endpoint.
+rpc-dial-url = "{{ consensus_execution_client_jsonrpc_endpoint }}"
+
+# Number of retries before shutting down consensus client.
+rpc-retries = "3"
+
+# RPC timeout for execution client requests.
+rpc-timeout = "2s"
+
+# Interval for the startup check.
+rpc-startup-check-interval = "3s"
+
+# Interval for the JWT refresh.
+rpc-jwt-refresh-interval = "20s"
+
+# Path to the execution client JWT-secret
+jwt-secret-path = "{{ consensus_jwt_secret_path }}"
+
```

Comment:
also in this file, I just noticed, please update suggested-fee-recipient = "0x0c6571537D0275Dbc6C93E00f4cA19500fa9c835"

and

graffiti = {{ org }}

Reply:
node api for validator has to be localhost ?

Reply:
Yes